Updates/4.12.4-b2.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Plugins\TagManager;
+
+use Piwik\Plugins\TagManager\Template\Tag\MatomoTag;
+use Piwik\Plugins\TagManager\Template\Variable\MatomoConfigurationVariable;
+use Piwik\Plugins\TagManager\Updates\NewVariableParameterMigrator;
+use Piwik\Updater;
+use Piwik\Updates as PiwikUpdates;
+
+/**
+ * Update for version 4.12.4-b1.
+ */
+class Updates_4_12_4_b3 extends PiwikUpdates
+{
+    public function doUpdate(Updater $updater)
+    {
+        $migrator = new NewTagParameterMigrator(MatomoTag::ID, 'customUrlRef');
+        $migrator->migrate();
+    }
+}
```

Comment:
Since you merged the changes from your other branch into this one, you don't have to create a new migration script. You should be able to add the two lines from the `doUpdate` method in this one to the end of the `doUpdate` in [the existing migration script](https://github.com/matomo-org/tag-manager/commit/1c52824eb4c0be4c35369376c31c22aea83f4f06#diff-28ece140c822d7c3d1db0e3bafb1d05e716b457a1b4e10e5912794378b532e82). That way both your variable and tag changes get migrated in the same script. I guess it doesn't hurt to have two separate scripts, but it makes the release process a little more complicated.

Reply:
@snake14 @tomper00 You should move this changes to `5.x-dev` branch since the next release is `5.x-dev`